src/commands/math/basicSymbols.ts

```diff
@@ -53,19 +57,19 @@ var DigitGroupingChar = P(Symbol, function(_, super_) {
     }
 
     // trim the leading spaces
-    while (right !== left && left.ctrlSeq === SPACE) {
+    while (right !== left && left && left.ctrlSeq === SPACE) {
```

Comment:
Minor, could simplify to left?.ctrlSeq

Reply:
Tracked as https://github.com/mathquill/mathquill/issues/984